src/App.tsx

```diff
@@ -1,14 +1,58 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
+import cn from 'classnames';
+import { getAllTodos } from './api/todos';
+import { Todo } from './types/Todo';
 import { UserWarning } from './UserWarning';
+import { Filters } from './utils/enums';
 
-const USER_ID = 0;
+const USER_ID = 10306;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedFilter, setSelectedFilter] = useState('All');
+  const [hasErrorMessage, setHasErrorMessage] = useState(true);
+  const [addError, setAddError] = useState(false);
+  const [deleteError] = useState(false);
+  const [updateError] = useState(false);
```

Comment:
Use `useState` with an initial value of false for `updateError` instead of a constant value.